### PR TITLE
Docker instructions

### PR DESCRIPTION
**Pull request by Calm-Rock**
### Summary

Docker tile added in environment of APM flows with instructions for each language and framework.

#### Related Issues / PR's

https://github.com/SigNoz/signoz.io/issues/1338

### CODE REVIEW

File: frontend/src/container/OnboardingContainer/Steps/EnvironmentDetails/EnvironmentDetails.tsx [hunk 1]
```diff
@@ -122,6 +126,18 @@ export default function EnvironmentDetails(): JSX.Element {
 					) {
 						return null;
 					}
+					if (
+						selectedModule?.id !== useCases.APM.id &&
+						environment.id === 'docker'
+					) {
+						return null;
+					}
+					if (
```
Comment (YounixM) [inline frontend/src/container/OnboardingContainer/Steps/EnvironmentDetails/EnvironmentDetails.tsx:135]: This looks same as the above if condition.

File: frontend/src/utils/app.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ export function extractDomain(email: string): string {
 
 export const isCloudUser = (): boolean => {
 	const { hostname } = window.location;
+	return true;
```
Comment (YounixM) [inline frontend/src/utils/app.ts:17]: Please remove this.

File: frontend/src/container/OnboardingContainer/Steps/EnvironmentDetails/EnvironmentDetails.tsx (hunk 1 above)
Comment (Calm-Rock) [reply frontend/src/container/OnboardingContainer/Steps/EnvironmentDetails/EnvironmentDetails.tsx:135]: Have added a Docker tile in the environment selection for APM only that's why this condition.
The above one is for **kubernetes**, this one is for **docker**.

Comment (YounixM) [reply frontend/src/container/OnboardingContainer/Steps/EnvironmentDetails/EnvironmentDetails.tsx:135]: There are 2 if blocks for docker

Comment (Calm-Rock) [reply frontend/src/container/OnboardingContainer/Steps/EnvironmentDetails/EnvironmentDetails.tsx]: Sorry. My bad. Fixed this.

**Review by YounixM — APPROVED** [review]

**Pull request merged**